test/core/simd/simd_align.wast

```diff
@@ -105,7 +105,7 @@
   (module quote
     "(memory 1) (func (drop (v128.load align=-1 (i32.const 0))))"
   )
-  "alignment must be a power of two"
+  "unknown operator"
```

Comment:
Seems like this test should be fixed to use a positive number, non-power-of-two instead.

Reply:
Ah yes, make sense. Let me review the tests cases to make sure they cover:
- negative -> unknown opreator
- positive non-power-of-two -> alignment error

Reply:
Hm, we do have test cases for positive non-power-of-two. For v128.load, we test for aligments -1, 0, 7.
I think we can remove the -1 case, or leave it to check for parse errors.

Reply:
I will leave these negative alignments here, it doesn't looked like we have such test cases anywhere else.